src/FlyWithLua.cpp

```diff
@@ -6684,8 +6679,7 @@ bool ReadAllScriptFiles()
                 // load the script file
                 std::ostringstream oss_ScrPathAndName;
                 oss_ScrPathAndName << scriptDir << "/" << FileToLoad;
-                std::string ScrPathAndName;
-                ScrPathAndName = oss_ScrPathAndName.str();
+                auto ScrPathAndName = oss_ScrPathAndName.str();
 
                 logMsg(logToDevCon, std::string("FlyWithLua Info: Start loading script file ").append(ScrPathAndName.c_str()));
```

Comment:
This could be written as `logMsg(logToDevCon, "FlyWithLua Info: Start loading script file " + ScrPathAndName);`. std::string overloads plus operator, so that you can concatenate it with a string literal or a C string.